source/css/_common/components/post/post-eof.styl

```diff
@@ -10,7 +9,7 @@
 }
 
 .post:last-child {
-  .post-eof.post-eof.post-eof {
```

Comment:
Screen? Where is this class?

Reply:
Not sure. But it same as `.post-eof`, no need to repeat 3 times

Reply:
Oh, my fault: i forgot to checkout your pull.

Reply:
This thing need to add delimeter between posts in homepage. And of course, last delimeter no needed, that's why display `none` on last child.

Reply:
But see, in Gemini I make same feature with another way:

https://github.com/theme-next/hexo-theme-next/blob/022b9ccb0ef1086b06039737e150dc27faae9e2c/source/css/_schemes/Gemini/index.styl#L45-L53

IDK which method is better, but I think need to choose only one.

Reply:
So, in Muse and Pisces – `.post-eof` enabled.
In Mist – disabled and in Gemini – disabled & implemented with another way.